tarynmartin/rancid-tomatillos
# Fix component structure

#### What’s this PR do?
- Create directories for each component
- Adjust imports on necessary files to work with new directory structure
- Add correct filepath for comment gif on README
#### Where should the reviewer start?
- Look at new directory set up in src
- Look at imports in index.js, App.js, Movies.js, MovieShow.js, and Comments.js
#### How should this be manually tested?
- in terminal, run npm start and look on browser to see functionality is maintained
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
